modules/checkout/tests/src/Functional/CheckoutOrderTest.php

```diff
@@ -158,8 +169,39 @@ public function testGuestOrderCheckout() {
     ], 'Continue to review');
     $this->assertSession()->pageTextContains('Contact information');
     $this->assertSession()->pageTextContains('Billing information');
+    $this->assertSession()->pageTextContains('Order Summary');
     $this->submitForm([], 'Pay and complete purchase');
     $this->assertSession()->pageTextContains('Your order number is 1. You can view your order on your account page when logged in.');
   }
 
+  /**
+   * Test OrderSummary settings.
+   */
+  public function testOrderSummarySettings() {
+    // Fill the cart.
+    $this->drupalGet($this->product->toUrl()->toString());
+    $this->submitForm([], 'Add to cart');
+
+    // Test OrderSummary default behavior.
```

Comment:
These two comments can be one. "Test the default settings: ensure the default view is shown".